src/NUnitEngine/nunit-agent/nunit-agent-x86.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="..\EngineVersion.cs" LinkBase="Properties" />
-    <Compile Include="..\nunit.engine\Internal\ExceptionHelper.cs" Link="ExceptionHelper.cs" />
+    <Compile Include="..\nunit.engine.core\Internal\ExceptionHelper.cs" Link="ExceptionHelper.cs" />
```

Comment:
It's tricky. The console runner would then have to directly reference the core engine directly. Otherwise we end up with duplication of the class in console tests. I followed that path for a bit, until it became too hairy and I reverted to shared source code for this particular helper.